plugins/inputs/nats_consumer/README.md

```diff
@@ -56,8 +56,13 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # password = ""
 
   ## Optional NATS 2.0 and NATS NGS compatible user credentials
+  ## Expects a credentials file with a JWT and nkey seed
   # credentials = "/etc/telegraf/nats.creds"
 
+  ## Optional NATS 2.0 Nkey seed file
+  ## Expects a file with the nkey seed
+  # credentials = "/etc/telegraf/seed.txt"
```

Comment:
Did you mean to call this "nkey_seed"? Does this depend on the credentials config option?

Reply:
Oh sorry, yes that is supposed to be `nkey_seed`.

No, it doesn't depend on the credentials config option.